.github/workflows/r_artifacts.yml

```diff
@@ -0,0 +1,34 @@
+name: R artifact builds
+
+on:
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  cran-package:
+    name: cran-package
+    if: github.event.comment.body == '/gha build r-artifacts' && contains('OWNER,MEMBER,COLLABORATOR', github.event.comment.author_association)
```

Comment:
/gha build r-artifacts

Reply:
ok this created this job: https://github.com/microsoft/LightGBM/actions/runs/332889387

which failed like this:

> No files were found with the provided path: /__w/LightGBM/LightGBM/lightgbm-3.0.0.99-r-cran.tar.gz\n. No artifacts will be uploaded.

Reply:
/gha build-r-artifacts

Reply:
/gha build r-artifacts

Reply:
this kicked off https://github.com/microsoft/LightGBM/actions/runs/332893223

which produced one artifact

![image](https://user-images.githubusercontent.com/7608904/97392727-7b7c8180-18b0-11eb-8b08-5d8cb266099b.png)

this creates a zip of the file, which is a little annoying, and it looks like you can't get around that (https://github.com/actions/upload-artifact#zipped-artifact-downloads):

> when a workflow run finishes and an artifact is downloaded from either the UI or through the download api, a zip is dynamically created with all the file contents that were uploaded. There is currently no way to download artifacts after a workflow run finishes in a format other than a zip or to download artifact contents individually

But I don't think it's too bad. So for releases, the workflow would be:

1. comment with `/gha build r-artifacts`
1. go to https://github.com/microsoft/LightGBM/actions, then to the artifacts action
1. once it's done running, download the .zip
1. unzip it, to get the artifact
1. manually upload artifacts to the release

Reply:
ok I think this is ready for review!

Reply:
I think this can be documented in https://github.com/microsoft/LightGBM/blob/master/R-package/README.md#preparing-a-cran-package.

Reply:
oh good idea. Added in https://github.com/microsoft/LightGBM/pull/3497/commits/bd895bfbd654ddc27f0ec31b08c45d2487602242